lib/vdsm/storage/blockVolume.py

```diff
@@ -428,6 +428,10 @@ def optimal_size(self):
             # minimal volume size.
             optimal_size = max(actual_size + MIN_PADDING, sc.MIN_CHUNK)
 
+        # Align to align size so callers can compare optimal size with the
```

Comment:
This reads a bit weird, maybe add single quotes around 'align size' or add the underscore to mimic the actual attribute name would make it clearer.

Reply:
You reviewed a patch which was part of #133, which is already merged now. I will post a cleanup for this comment separately since this change is not part of this PR.